docs/UserGuide.md

```diff
@@ -238,6 +238,7 @@ Format: `find parameter/KEYWORD [parameter/KEYWORD]...`
 * The search is case-insensitive. E.g., `bobby` will match `Bobby`.
 * Applicants matching at least one keyword will be returned (i.e., OR search). E.g., `n/Bobby Cortez` will return applicants with the name `Bobby Lacruz`, `Alexander Cortez`
 * The order of the keywords does not matter. E.g., `Bobby cortez` will match `Cortez bobby`.
+* Multiple field search: Across different fields, an _AND_ search is done, while an _OR_ search is still done within each field for its keywords. E.g., `n\Bobby Cortez g/male` will return male applicants with names matching `Bobby` or `Cortez`.
```

Comment:
Will it be better if we standardise the slash to be `/`?

Reply:
You are right. It was a typo on my end. Thanks for spotting it